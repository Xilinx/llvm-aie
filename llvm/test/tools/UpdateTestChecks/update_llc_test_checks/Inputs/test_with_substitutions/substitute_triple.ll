; RUN: llc -mtriple=%triple < %s | FileCheck %s

define i64 @i64_test(i64 %i) nounwind readnone {
  ret i64 %i
}
