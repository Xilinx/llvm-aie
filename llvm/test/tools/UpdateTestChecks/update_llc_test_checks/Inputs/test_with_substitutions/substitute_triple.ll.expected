; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
; RUN: llc -mtriple=%triple < %s | FileCheck %s

define i64 @i64_test(i64 %i) nounwind readnone {
; CHECK-LABEL: i64_test:
; CHECK:       # %bb.0:
; CHECK-NEXT:    movq %rdi, %rax
; CHECK-NEXT:    retq
  ret i64 %i
}
